OpenSim/Actuators/VandenBogert2011Muscle.cpp

```diff
@@ -0,0 +1,1234 @@
+/* -------------------------------------------------------------------------- *
+ *                      OpenSim:  VandenBogert.cpp                            *
+ * -------------------------------------------------------------------------- */
+
+
+//=============================================================================
+// INCLUDES
+//=============================================================================
+#include "VandenBogert2011Muscle.h"
+#include <OpenSim/Simulation/Model/Model.h>
+
+
+//=============================================================================
+// STATICS
+//=============================================================================
+using namespace std;
+using namespace OpenSim;
+
+
+
+//=============================================================================
+// CONSTRUCTOR(S) AND DESTRUCTOR
+//=============================================================================
+//_____________________________________________________________________________
+/*
+ * Default constructor
+ */
+VandenBogert2011Muscle::VandenBogert2011Muscle()
+{
+    constructProperties();
+}
+
+//_____________________________________________________________________________
+/*
+ * Constructor.
+ */
+
+VandenBogert2011Muscle::VandenBogert2011Muscle(const std::string& aName)
+{
+    setName(aName);
+    constructProperties();
+}
+
+//_____________________________________________________________________________
+/*
+ * Construct and initialize properties.
+ * All properties are added to the property set. Once added, they can be
+ * read in and written to files.
+ */
+void VandenBogert2011Muscle::constructProperties()
+{
+    setAuthors("A. van den Bogert, B. Humphreys, C. Dembia");
```

Comment:
Added: 
van den Bogert A, Blana D, Heinrich D, Implicit methods for efficient musculoskeletal simulation and optimal control. Procedia IUTAM 2011; 2:297-316